src/utils/enableActions.ts

```diff
@@ -0,0 +1,23 @@
+import { Octokit } from "./octokitTypes";
+import { inform } from "./globals";
+
+export const enableActions = async (
+  owner: string,
+  repo: string,
+  octokit: Octokit
+): Promise<void> => {
+  const { status } = await octokit.request(
+    "PUT /repos/{owner}/{repo}/actions/permissions",
+    {
+      owner,
+      repo,
+      enabled: true,
+    }
+  );
+
+  if (status !== 204) {
```

Comment:
What happens if actions are already enabled? 🤔 

Does this throw a `204` even if actions are already enabled?

Reply:
Yes, `204` even if actions are already enabled.